packages/eas-cli/src/graphql/types/Update.ts

```diff
@@ -9,6 +9,7 @@ export const UpdateFragmentNode = gql`
     runtimeVersion
     platform
     manifestFragment
+    gitCommitHash
```

Comment:
This also fixes an issue where we "generate" a new git commit hash when republishing older updates.